### PR TITLE
[IMP] base_report_to_printer: archive printers

### PR DESCRIPTION
**Pull request by jarroyomorales**
Cherry-pick from #232 

@etobella @bodedra

### CODE REVIEW

File: base_report_to_printer/views/printing_printer.xml [hunk 1]
```diff
@@ -33,6 +33,20 @@
...
+                    <div class="oe_button_box" name="button_box">
+                        <button
+                            name="toggle_active"
+                            type="object"
+                            class="oe_stat_button"
+                            icon="fa-archive"
+                        >
+                            <field
+                                name="active"
+                                widget="boolean_button"
```
Comment (etobella) [inline base_report_to_printer/views/printing_printer.xml]: Boolean button does not work on 13.0

**Review by etobella — CHANGES_REQUESTED** [review]

File: base_report_to_printer/views/printing_printer.xml (hunk 1 above)
Comment (jarroyomorales) [reply base_report_to_printer/views/printing_printer.xml]: True, fixed according to the new paradigm.

**Review by etobella — APPROVED** [review]

**Review by alexis-via — APPROVED** [review]

**Comment by alexis-via** [comment]
Pedro's remark on the other PR seems important : 
<<
Shouldn't this be taken into account when updating printers from server (with active_test=False or similar for doing the match)?
>>

**Comment by alexis-via** [comment]
It seems there is nothing to change, because in the method update_printers() on printing.server, the loop uses "server.printer_ids" (the O2M field), so it will also take into account the inactive printers.

**Comment by jarroyomorales** [comment]
@alexis-via I tested the same on v13 and as I said in https://github.com/OCA/report-print-send/pull/232#issuecomment-745178190 we need to add the `active_test` context, since `server.printer_ids` doesnt return the archived printers.

Thank you for the review!

**Comment by pedrobaeza** [comment]
Please fix pre-commit

**Comment by jarroyomorales** [comment]
@pedrobaeza Its fixed here https://github.com/OCA/report-print-send/pull/240

**Comment by alexis-via** [comment]
You're right server.printer_ids doesn't return the inactive printers (I was excepting the opposite !). I updated the v14 code too.

**Pull request merged**